dcos/rpcclient.py

```diff
@@ -145,7 +144,8 @@ def http_req(self, method_fn, path, *args, **kwargs):
                 request_method=e.response.request.method,
                 request_url=e.response.request.url,
                 json_body=json_body)
-            raise DCOSException(message)
+
+            raise DCOSException(message) from e
```

Comment:
Hehe, I think we want the same here. I'm all for hiding stack trace unless debug mode is enabled. I'm not sure how that applies to `raise DCOSException(message) from e` though.